renderdoc/driver/gl/gl_replay.cpp

```diff
@@ -206,9 +207,8 @@ void GLReplay::CreateOutputWindowBackbuffer(OutputWindow &outwin, bool depth)
   gl.glGenTextures(1, &outwin.BlitData.backbuffer);
   gl.glBindTexture(eGL_TEXTURE_2D, outwin.BlitData.backbuffer);
 
-  gl.glTextureImage2DEXT(outwin.BlitData.backbuffer, eGL_TEXTURE_2D, 0, eGL_SRGB8, outwin.width,
-                         outwin.height, 0, eGL_RGBA, eGL_UNSIGNED_BYTE, NULL);
-  gl.glTexParameteri(eGL_TEXTURE_2D, eGL_TEXTURE_MAX_LEVEL, 0);
+  gl.glTextureStorage2DEXT(outwin.BlitData.backbuffer, eGL_TEXTURE_2D, 1, eGL_SRGB8, outwin.width,
+                           outwin.height);
```

Comment:
I can change it back, however the original arguments of `glTextureImage2DEXT` calls caused errors in case of GLES (`invalid operation error` and `<format> or <internalFormat> not valid`). According to the specs of `glTextureImage2D` if `internalFormat` is `GL_SRGB8` than the `format` and `type` parameters must be `GL_RGB` and `GL_UNSIGNED_BYTE`. Similarly for `GL_DEPTH_COMPONENT24` these must be `GL_DEPTH_COMPONENT` and `GL_UNSIGNED_INT`. If I modify the code accordingly it seems to work for both GL and GLES.